lib/rbvmomi/logging.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module RbVmomi
+  class << self
+    attr_writer :logger
+  end
+
+  def self.logger
+    require 'logger'
+    @logger ||= Logger.new($stderr)
+  end
+
+  module Logging
+    def logger
+      RbVmomi.logger
+    end
+  end
```

Comment:
What is this part for?  I don't see calls to Logger, nor something that mixes in Logging.

Reply:
Yeah we can drop this, is just convenience to not have to call out the full `RbVmomi.logger` if we wanted to `include Logging` in a class